Gemfile.jruby-1.9.lock

```diff
@@ -724,4 +724,4 @@ DEPENDENCIES
   rspec (~> 3.1.0)
   rubyzip (~> 1.1.7)
   simplecov
-  stud (~> 0.0.19)
+  stud (~> 0.0.21)
```

Comment:
we should force this to be ~> 0.0.22, that's where the new stud stop stuff was added